snapshotter/service_integ_test.go

```diff
@@ -49,15 +49,15 @@ const (
 )
 
 func TestLaunchContainerWithRemoteSnapshotter_Isolated(t *testing.T) {
-	integtest.Prepare(t, integtest.WithDefaultNetwork())
```

Comment:
Not needed. All `CreateVM` calls specify network interface.

Reply:
Let's have this one in either a different commit and/or PR.

Reply:
Question regarding this: have you tried just making this change and observed any different behavior on the BuildKite tests?